### ISSUE TITLE
Embedded user map

### ISSUE THREAD

**Issue by kachkaev** [effrenus/yandex-map-react#9]
I guess that the most common potential use of the component is an embedded map on a company's contacts page. These maps are sometimes drawn as a user map and can be simply added as an overlay:

``` js
var ml = new YMaps.YMapsML('http://maps.yandex.ru/export/usermaps/xxx/');
map.addOverlay(ml);
```

It would be nice to have this functionality in the React version too:

``` js
<Map center={ [55.754734, 37.583314] } zoom={ 10 } >
  <UserMapLayer id="xxx" onLoad={ this._userMapLoad } onLoadError={ this._userMapLoadError } />
</Map>
```

How difficult would this be to add? What alternatives to YMapsML exist in 2016? I might be pretty outdated about the way the things are done these days since I haven't been implementing anything YMaps-related for about 5 yrs. YMapsML seemed to be a good way of showing the driving directions to a company with no need to program the markers, the territories and the arrows manually.

**Comment by effrenus** [comment]
Yes, it would be handy) 
"My maps" is closed in favour "[Map constructor](https://tech.yandex.com/maps/tools/constructor/)", that doesn't support export yet. But such functionality in plans. I'll ask about support of [YMapsML](https://tech.yandex.com/maps/doc/ymapsml/1.x/guide/concepts/About-docpage/) as export format and other variants.

**Comment by kachkaev** [comment]
Got it. How would you recommend to display something like this on the map?
https://yandex.github.io/mapsapi-examples-old/html/visualisationymapsml.html
(this is from [API v1 docs](https://tech.yandex.ru/maps/doc/jsapi/1.x/dg/concepts/ymapsml-docpage))

It does not have to be MapML per se, just wondering what else can substitute [what I've already got](http://maps.yandex.ru/-/CRCGQ5b).

**Comment by effrenus** [comment]
It's possible, https://github.com/effrenus/yandex-map-react-examples/tree/usermap, but a little bit tricky)

In readme I've described the process.

**Comment by kachkaev** [comment]
Looks promising! I'll try it tonight.

**Comment by kachkaev** [comment]
It worked, many thanks! I'll keep the issue open just if you want to improve the link to the constructor. Feel free to close it if you don't have such plans – my problem is solved!

**Comment by kachkaev** [comment]
BTW there is still an issue with user maps / coordorder. If `coordorder="longlat"` is not specified for the `<Map />`, the user map becomes flipped. Mine moved from Penza to Kazakhstan :–)

![screen shot 2016-05-19 at 22 00 30](https://cloud.githubusercontent.com/assets/608862/15409568/348f8274-1e0d-11e6-886c-7b3ea7924bda.png)

↓

![screen shot 2016-05-19 at 21 30 59](https://cloud.githubusercontent.com/assets/608862/15409538/0e7d6ea2-1e0d-11e6-8797-fb736906dc54.png)

**Comment by effrenus** [comment]
Yes, you are right. Default coordinate order is `latlong`.  But coordinates in constructor stored in `longlat` order(
